ocs_ci/ocs/platform_nodes.py

```diff
@@ -2225,6 +2225,31 @@ def stop_nodes(self, nodes, timeout=600, wait=True, force=True):
                 node_names=node_names, status=constants.NODE_NOT_READY, timeout=timeout
             )
 
+    def restart_nodes(self, nodes, timeout=900, wait=True, force=True):
+        """
+        Restart RHV VM
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            timeout (int): time in seconds to wait for node to reach 'ready' state
+            wait (bool): True if need to wait till the restarted node reaches
+                READY state. False otherwise
+            force (bool): True for force VM reboot, False otherwise
+
+        """
+        if not nodes:
+            raise ValueError("No nodes found for restarting")
```

Comment:
This ValueError should be documented in the docstring under `Raises:` section.

Reply:
incoperate the review comment